wiki/People/The_Team/Account_support_team/id.md

```diff
@@ -25,17 +20,24 @@ Kamu harus menghubungi tim ini mengenai topik yang berhubungan dengan akun yang
 
 ### [accounts@ppy.sh](mailto:accounts@ppy.sh)
 
-- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk silence dan penghapusan konten profil.
+- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk:
```

Comment:
iirc all instances of `pembatasan` in the osu! website has been reverted back to `restriction` for convenience-related purposes tho. 

if you want to keep the `pembatasan` here you might want to go for sth like `Banding atas pembatasan (*restriction*) yang diberlakukan`

![image](https://user-images.githubusercontent.com/36564236/107544554-ee5dfe00-6bfc-11eb-8ed7-88f47a515acd.png)

Reply:
Changed `pembatasan` to `restriction` on line 5 and 17. I'll take a look for this article https://osu.ppy.sh/wiki/id/Help_Centre/Account_Restrictions later, as well as in other articles that contain restrictions.